### ISSUE TITLE
SVG element

### ISSUE THREAD

**Issue by anthonator** [jozo/all-html-elements-and-attributes#4]
I noticed that the `svg` element is missing from `html-elements.json`. I dug around MDN and noticed they have a separate set of reference docs for SVG (linked below). Would you consider adding the `svg` element or do you think this is out of scope?

MDN SVG element reference:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element

**Comment by artyuum** [comment]
Indeed, the `svg` element is missing. No clue why it's not referenced in the [dedicated HTML page](https://developer.mozilla.org/en-US/docs/Web/HTML), but on the SVG page it says the following so that's probably why:
> Scalable Vector Graphics (SVG) is an [XML](https://developer.mozilla.org/en-US/docs/Web/XML)-based markup language for describing two-dimensional based [vector graphics](https://en.wikipedia.org/wiki/Vector_graphics).

@jozo I can make a PR add the `svg` element in the `html-elements.json` file with a separate json file for the specific SVG elements (`svg-elements.json`) unless you believe this is out of scope.

WDYT?
